mteb/run_task/run_tasks.py

```diff
@@ -104,29 +102,38 @@ def _run_task(
     return result
 
 
-def run_task(
+def run_tasks(
     model: Encoder | SentenceTransformer | CrossEncoder,
-    task: AbsTask,
+    task: AbsTask,  # TODO: allow multiple tasks + benchmarks
     *,
-    splits: list[str] | None = None,
-    subsets_to_run: list[str] | None = None,
     co2_tracker: bool | None = None,
     raise_error: bool = True,
     encode_kwargs: dict[str, Any] | None = None,
-    output_folder: Path | str | None = None,
+    output_folder: Path | str | None = "results",
+    cache_strategy: Literal[
+        "no_cache", "output_folder", "online_cache", "only_cache"
+    ] = "output_folder",
+    overwrite_strategy: Literal["always", "never", "only-missing"] = "only-missing",
     **kwargs: Any,
 ) -> TaskResult | None:
     """This function runs a model on a a given task and returns the results.
 
     Args:
         model: The model to use for encoding.
-        tasks: A task to run.
-        splits: The splits to evaluate on. If None, the default evaluation splits for the task will be used.
-        subsets_to_run: The subsets to evaluate on. If None, all subsets will be evaluated.
+        task: A task to run.
         co2_tracker: If True, track the CO₂ emissions of the evaluation. If none is passed co2 tracking will be run if codecarbon is installed.
         encode_kwargs: Additional keyword arguments passed to the models `encode` method.
         raise_error: If True, raise an error if the task fails. If False, return an empty list.
         output_folder: The folder to save the results to. If None, the results will not be saved.
+        cache_strategy: The strategy to use for loading existing the results. Can be:
```

Comment:
`Cache` can be confused with embeddings cache 
```suggestion
        results_cache_strategy: The strategy to use for loading existing the results. Can be:
```